.evergreen/config.yml

```diff
@@ -671,6 +1090,10 @@ axes:
         display_name: "latest"
         variables:
            MONGODB_VERSION: "latest"
+      - id: "4.4"
```

Comment:
Hmm, I thought this was already added; are you fully rebased with the latest master?

Reply:
Yep, this axis was actually not used by our testing at all (until now). Instead, we have separate task definitions for each of the different server versions. The OCSP ones introduce a ton of new tasks per version so it made sense for them to leverage the axis instead of being bundled together, at least from the `config.yml`'s perspective.

Reply:
Ah, right, I misread what this axis was